### ISSUE TITLE
Dependabot keeps spamming my forked repo into oblivion even though it is disabled

### ISSUE THREAD

**Issue by siepkes** [dependabot/dependabot-core#4291]
Exhibit A:

![dependabot_2](https://user-images.githubusercontent.com/1428678/136335800-b17c5363-df6e-4c2f-9659-b6d2a39df751.png)

Exhibit B:

![dependabot_1](https://user-images.githubusercontent.com/1428678/136335802-fa786dec-835b-432a-bd64-696fd3cba20f.png)

**Comment by jurre** [comment]
Yeah agreed, definitely room for improvement on how we handle forks! Most of that is tracked here https://github.com/dependabot/dependabot-core/issues/2198, so I'm going to close this out for now.
